src/wallet/types.ts

```diff
@@ -373,6 +373,56 @@ export interface FullNodeVersionData {
   maxNumberOutputs: number;
 };
 
+export interface PushNotificationResponseData {
+  success: boolean,
+  error?: string,
+  /** This property shows up in case of validation error. */
```

Comment:
This comment references which code line?

Reply:
The `details?` bellow.